Framework/PythonInterface/mantid/kernel/src/Registry/SequenceTypeHandler.cpp

```diff
@@ -87,7 +87,8 @@ SequenceTypeHandler<ContainerType>::create(const std::string &name, const boost:
 
   ContainerType valueInC;
   // Current workaround for things that still pass back wrapped vectors...
-  if (boost::starts_with(defaultValue.ptr()->ob_type->tp_name, "std_vector")) {
+  std::string tpName = defaultValue.ptr()->ob_type->tp_name;
```

Comment:
Can the `tpName`'s in this file be const?

Reply:
Yes, that's quite right. I probably just introduced a future cppcheck warning there, I will make them const.